Resources/config/services.yml

```diff
@@ -9,5 +10,6 @@ services:
     Chaplean\Bundle\OvhLogsBundle\Processor\OvhLogsProcessor:
         arguments:
             $token: '%ovh_logs.token%'
+            $env: '%ovh_logs.env%'
```

Comment:
If you really wanted to pass environment, why we need to set it again instead of using current environment of project?